spec/features/sign_ups_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.feature "SignUps", type: :feature do
+  	scenario 'with valid email and password' do
+    sign_up_with 'valid@example.com', 'password'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.